### ISSUE TITLE
Error 400 when forwarding to Jenkins

### ISSUE THREAD

**Issue by MartiBolet** [chmouel/gosmee#68]
Hi, I'm using gosmee to pass github webhooks to my jenkins server. I'm facing an HTTP code 400 when the message is passed to Jenkins directly, but when i save the payload and i replay it with the bash script autogenerated with `--saveDir` it works with HTTP code 200 and Jenkins triggers correctly.

Info: 
I'm running gosmee from docker:
`docker run -d --net host --name gosmee ghcr.io/chmouel/gosmee:latest client --saveDir /home/gosmee https://hook.pipelinesascode.com/<my_id> http://localhost:8080/github-webhook/`
Result of `docker logs gosmee`:
```
Starting gosmee version: dev
Forwarding https://hook.pipelinesascode.com/<my_id> to http://localhost:8080/github-webhook/
/home/gosmee/push-20230322T15h49.sh and /home/gosmee/push-20230322T15h49.json has been saved
push event replayed to http://localhost:8080/github-webhook/, status: 400, error: 400 Bad Request
```
And when i execute `/home/gosmee/push-20230322T15h49.sh` jenkins is triggered and works fine. I'm doing something wrong?

Thank you,

**Comment by chmouel** [comment]
that's weird, i haven't tried jenkins, gosmee set exactly the same things as what the curl generated script is doing..

**Comment by chmouel** [comment]
@MartiBolet did you find the issue?

**Comment by MartiBolet** [comment]
No, I didn't but I don't have time to look into it right now so I'm closing it to don't leave an open issue when maybe it's my jenkins configuration.

**Comment by chmouel** [comment]
no worries! if i get a chance to have access to a jenkins webhook then i'll see what i can do.. 

something i can think of now is to increase the timeout of this flag and see if it works.
```
--target-connection-timeout value                                  How long to wait when forwarding the request to the service (default: 5) [$GOSMEE_TARGET_TIMEOUT]
```

**Comment by chmouel** [comment]
thanks to @portante it seems the bug was because we didn't set properly the content-type, hopefully it should work now! and should be released in 0.16.0
